deis-workflow-manager/manifests/deis-workflow-manager-rc.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-workflow-manager
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  replicas: 1
+  selector:
+    app: deis-workflow-manager
+  template:
+    metadata:
+      labels:
+        app: deis-workflow-manager
+    spec:
+      serviceAccount: deis-workflow-manager
+      containers:
+      - name: deis-workflow-manager
+        image: quay.io/deis/workflow-manager:v2-beta
```

Comment:
It'd probably make sense to set this to `quay.io/deisci/workflow-manager:v2-beta` (note the `ci` bit).